rules/jms/tests/Rector/ClassMethod/RemoveJmsInjectParamsAnnotationRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\JMS\Tests\Rector\ClassMethod\RemoveJmsInjectParamsAnnotationRector\Fixture;
+
+use JMS\DiExtraBundle\Annotation as DI;
+
+class SomeClass
+{
+    /**
+     * @DI\InjectParams({
+     *     "subscribeService" = @DI\Inject("app.email.service.subscribe"),
+     *     "ipService" = @DI\Inject("app.util.service.ip")
+     * })
+     */
+    public function __construct() {
```

Comment:
Shouldn't this be (maybe not before, but after for sure if you use i.e. Symfony):
```php
public function __construct($subscribeService, $ipService)
{
}

Reply:
It's only about doc blocks. The code is minimal possible, see the original issue